tests/views/test_collection_exercise.py

```diff
@@ -594,10 +592,9 @@ def test_collection_exercise_view_seft_validated(self, mock_request, mock_detail
         self.assertEqual(response.status_code, 200)
         self.assertIn("Monthly Survey of Building Materials Bricks".encode(), response.data)
         self.assertIn("074".encode(), response.data)
-        self.assertIn("Sample loaded".encode(), response.data)
-        self.assertIn("test_collection_instrument.xlxs".encode(), response.data)
+        self.assertIn("Uploaded".encode(), response.data)
```

Comment:
Once I see a block of code 3 time, I am thinking we need to extract it. If the dev before did this, it would have saved you the pain of doing it for each one

Reply:
To be done in another card